src/test/java/org/dinosaur/foodbowl/domain/photo/application/PhotoLocalUploaderTest.java

```diff
@@ -0,0 +1,73 @@
+package org.dinosaur.foodbowl.domain.photo.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.List;
+import org.dinosaur.foodbowl.file.FileTestUtils;
+import org.dinosaur.foodbowl.global.exception.FileException;
+import org.dinosaur.foodbowl.test.IntegrationTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.web.multipart.MultipartFile;
+
+@SuppressWarnings("NonAsciiCharacters")
+class PhotoLocalUploaderTest extends IntegrationTest {
+
+    @Autowired
+    private PhotoLocalUploader fileSystemPhotoService;
+
+    @Test
+    void 파일을_디렉토리에_저장한다() {
+        MockMultipartFile multipartFile = new MockMultipartFile("bucket", "foodBowl.jpg", MediaType.TEXT_PLAIN_VALUE,
+                "Hello Images".getBytes());
+        List<MultipartFile> multipartFiles = List.of(multipartFile);
+
+        List<String> filePaths = fileSystemPhotoService.upload(multipartFiles, "test");
+
+        assertThat(filePaths).isNotEmpty();
+        FileTestUtils.cleanUp();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"foodBowl.zip", "foodBowl.pdf", "foodBowl.docx", "foodBowl.pptx"})
+    void 이미지_파일이_아니면_예외가_발생한다(String originalFilename) {
+        MockMultipartFile multipartFile = new MockMultipartFile("bucket", originalFilename, MediaType.TEXT_PLAIN_VALUE,
+                "Hello Images".getBytes());
+        List<MultipartFile> multipartFiles = List.of(multipartFile);
+
+        assertThatThrownBy(() -> fileSystemPhotoService.upload(multipartFiles, "test"))
+                .isInstanceOf(FileException.class)
+                .hasMessage("이미지 파일만 업로드 가능합니다.");
+    }
+
+    @ParameterizedTest
+    @NullAndEmptySource
+    @ValueSource(strings = {" "})
```

Comment:
```suggestion
    @NullAndEmptySource
    @ValueSource(strings = {" "})
    @ParameterizedTest
```
`ditto`

`@Source`를 여러 개 사용할 수 있나요???

Reply:
넹 사용할 수 있더라구욥

![image](https://github.com/Busan-Dinosaur/Foodbowl-Backend/assets/68818952/29fe6e6c-87d4-43e6-bb72-566290b47e5e)